client/components/theme/more-button.jsx

```diff
@@ -20,15 +20,33 @@ var PopoverMenu = require( 'components/popover/menu' ),
  */
 var ThemeMoreButton = React.createClass( {
 	propTypes: {
-		// Theme ID (theme-slug)
-		id: React.PropTypes.string.isRequired,
-		// Index of theme in results list, if any
+		theme: React.PropTypes.shape( {
```

Comment:
Yeah, I'd like to see a Calypso lib of PropTypes somewhere. Maybe something following the `client/state` hierarchy?